src/site/antora/modules/ROOT/pages/faq.adoc

```diff
@@ -17,404 +17,196 @@ Licensed to the Apache Software Foundation (ASF) under one or more
-        <artifactId>log4j</artifactId>
-      </exclusion>
-      <exclusion>
-        <groupId>org.slf4j</groupId>
-        <artifactId>slf4j-log4j12</artifactId>
-      </exclusion>
-    </exclusions>
-  </dependency>
-  <dependency>
-    <groupId>org.apache.logging.log4j</groupId>
-    <artifactId>log4j-core</artifactId>
-    <version>${Log4jReleaseVersion}</version>
-  </dependency>
-  <dependency>
-    <groupId>org.apache.logging.log4j</groupId>
-    <artifactId>log4j-slf4j-impl</artifactId>
-    <version>${Log4jReleaseVersion}</version>
-  </dependency>
-  <dependency>
-    <groupId>org.apache.logging.log4j</groupId>
-    <artifactId>log4j-1.2-api</artifactId>
-    <version>${Log4jReleaseVersion}</version>
-  </dependency>
-</dependencies>
-----
-
-Dependencies can be globally excluded in Gradle like so:
-
-[source]
-----
-configurations {
-  all*.exclude group: 'log4j', module: 'log4j'
-}
-----
-
-The equivalent Gradle config for the above Maven exclusion would look like:
-
-[source]
-----
-dependencies {
-  compile('com.example:example-project:1.0') {
-    exclude group: 'log4j', module: 'log4j'
-    exclude group: 'org.slf4j', module: 'slf4j-log4j12'
-  }
-  compile('org.apache.logging.log4j:log4j-core:${Log4jReleaseVersion}')
-  compile('org.apache.logging.log4j:log4j-slf4j-impl:${Log4jReleaseVersion}')
-  compile('org.apache.logging.log4j:log4j-1.2-api:${Log4jReleaseVersion}')
-}
-----
+This page compiles a list of frequently asked questions.
+If you don't find the answer to your question, please consult to link:{logging-services-url}/support.html[the support page].
 
 [#config_location]
 == How do I specify the configuration file location?
 
-By default, Log4j looks for a configuration file named *log4j2.xml* (not log4j.xml) in the classpath.
+By default, Log4j looks for a configuration file named `log4j2.xml`, `log4j2.properties`, etc. in the classpath.
+
+[WARNING]
+====
+Log4j 1 (which has reached its end-of-life in 2015!) uses `log4j.xml`.
+Log4j 2 and onwards use `log4j2.xml`.
+====
 
-You can also specify the full path of the configuration file with this system property: `-Dlog4j.configurationFile=path/to/log4j2.xml`
+You can also specify the full path of the configuration file using a system property: +
+`-Dlog4j.configurationFile=/path/to/log4j2.xml`
 
 That property can also be included in a classpath resource file named `log4j2.component.properties`.
 
 Web applications can specify the Log4j configuration file location with a servlet context parameter.
-See http://logging.apache.org/log4j/2.x/manual/webapp.html#ContextParams[this section] of the Using Log4j 2 in Web Applications manual page.
+See xref:manual/webapp.adoc#ContextParams[the related page on web applications].
+
+Refer to xref:manual/configuration.adoc[the Configuration page] for further details.
 
 [#config_from_code]
-== How do I configure log4j2 in code without a configuration file?
+== How do I configure Log4j programmatically?
```

Comment:
@ppkarwasz, there are several sections here dealing with programmatic configuration. I need your help to wire these to the configuration pages you are working on.

Reply:
I haven't gotten to that part yet, you can add some bogus link and I'll replace it later.